infrastructure/scripts/smoke-test.sh

```diff
@@ -29,29 +29,43 @@ function apiCall {
   echo ${response_payload}
 }
 
-login_payload=$(apiCall "users.login" '{"email":"grace@example.com","password":"the_answer_is_42"}' 200)
+function getValueFrom {
+  local payload=$(tail -1 <<< "$1")
+  echo $payload | jq -r ".$2"
+}
+
+login_response=$(apiCall "users.login" '{"email":"grace@example.com","password":"the_answer_is_42"}' 200)
+
+token=$(echo $login_response | jq -r '.token')
+
+channels_connect_response=$(apiCall "channels.connect" '{"source": "chat_plugin", "source_channel_id": "my-chat-channel-1", "token": "wat", "name": "chat plugin source", "image_url": ""}' 200 ${token})
+channels_list=$(apiCall "channels.list" '{}' 200 ${token})
+channels_explore=$(apiCall "channels.explore" '{}' 200 ${token})
+channel_id=$(getValueFrom $channels_connect_response "id")
+
+
+chatplugin_authenticate_response=$(apiCall "chatplugin.authenticate" "{\"channel_id\": \"$channel_id\"}" 200 ${token})
+chatplugin_token=$(getValueFrom $chatplugin_authenticate_response "token")
+
+chatplugin_send_response=$(apiCall "chatplugin.send" '{"message": {"text": "Message from chatplugin"}}' 200 ${chatplugin_token})
 
-token=$(echo $login_payload | jq -r '.token')
-conversation_list_payload=$(apiCall "conversations.list" '{}' 200 ${token})
-conversations_info_payload=$(apiCall "conversations.info" '{}' 200 ${token})
-conversations_read_payload=$(apiCall "conversations.read" '{}' 200 ${token})
+conversation_list_response=$(apiCall "conversations.list" '{}' 200 ${token})
+conversation_id=$(getValueFrom $conversation_list_response "data[0].id")
+conversations_info_response=$(apiCall "conversations.info" "{\"conversation_id\": \"$conversation_id\"}" 200 ${token})
+conversations_read_response=$(apiCall "conversations.read" "{\"conversation_id\": \"$conversation_id\"}" 202 ${token})
 
-chatplugin_authenticate_payload=$(apiCall "chatplugin.authenticate" '{}' 200 ${token})
 
-channels_list_payload=$(apiCall "channels.list" '{}' 200 ${token})
-channels_disconnect_payload=$(apiCall "channels.disconnect" '{}' 200 ${token})
-channels_explore_payload=$(apiCall "channels.explore" '{}' 200 ${token})
-channels_connect_payload=$(apiCall "channels.connect" '{}' 200 ${token})
+messages_send_response=$(apiCall "messages.send" "{\"conversation_id\": \"$conversation_id\", \"message\":{\"text\": \"Response from Airy\"}}" 200 ${token})
+messages_list_response=$(apiCall "messages.list" "{\"conversation_id\": \"$conversation_id\"}" 200 ${token})
```

Comment:
as we have  a message in the conversation, I think we can also check the response contains the right message